File: Data Structures & Algorithms/number-of-one-bits/submission-0.cpp
class Solution {
public:
    int hammingWeight(uint32_t n) {
        uint32_t i{n};
        unsigned int fin{0};
        for (int z{0}; z < 32; ++z) {
            if (i & 1) {
                ++fin;
            }
            i >>= 1;
        }
        return fin;
    }
};

File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        fin = defaultdict(list)
        for s in strs:
            cnt = 26 * [0]
            for c in s:
                cnt[ord(c) - ord("a")] += 1
            fin[tuple(cnt)].append(s)
        return list(fin.values())